File: SiriusQuality/SiriusQuality-Priestly-Taylor/src/java/SiriusQuality_Priestly_Taylor/PotentialET_PTAuxiliary.java
import  java.io.*;
import  java.util.*;
import java.time.LocalDateTime;

public class PotentialET_PTAuxiliary
{
    private double solarRadiation;
    private double hslope;
    private double netRadiation;
    
    public PotentialET_PTAuxiliary() { }
    
    public PotentialET_PTAuxiliary(PotentialET_PTAuxiliary toCopy, boolean copyAll) // copy constructor 
    {
        if (copyAll)
        {
            this.solarRadiation = toCopy.getsolarRadiation();
            this.hslope = toCopy.gethslope();
            this.netRadiation = toCopy.getnetRadiation();
        }
    }
    public double getsolarRadiation()
    { return solarRadiation; }

    public void setsolarRadiation(double _solarRadiation)
    { this.solarRadiation= _solarRadiation; } 
    
    public double gethslope()
    { return hslope; }

    public void sethslope(double _hslope)
    { this.hslope= _hslope; } 
    
    public double getnetRadiation()
    { return netRadiation; }

    public void setnetRadiation(double _netRadiation)
    { this.netRadiation= _netRadiation; } 
    
}

File: SiriusQuality/SiriusQuality-Priestly-Taylor/src/cpp/SiriusQuality_Priestly_Taylor/PotentialET_PTAuxiliary.cpp
#include "PotentialET_PTAuxiliary.h"
using namespace SiriusQuality_Priestly_Taylor;

PotentialET_PTAuxiliary::PotentialET_PTAuxiliary() {}

double PotentialET_PTAuxiliary::getsolarRadiation() { return this->solarRadiation; }
double PotentialET_PTAuxiliary::gethslope() { return this->hslope; }
double PotentialET_PTAuxiliary::getnetRadiation() { return this->netRadiation; }

void PotentialET_PTAuxiliary::setsolarRadiation(double _solarRadiation) { this->solarRadiation = _solarRadiation; }
void PotentialET_PTAuxiliary::sethslope(double _hslope) { this->hslope = _hslope; }
void PotentialET_PTAuxiliary::setnetRadiation(double _netRadiation) { this->netRadiation = _netRadiation; }